src/routes/__tests__/DisciplineSelectionScreen.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import { RouteProp } from '@react-navigation/native'
+import { fireEvent, waitFor } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+
+import { useLoadDisciplines } from '../../hooks/useLoadDisciplines'
+import useReadSelectedProfessions from '../../hooks/useReadSelectedProfessions'
+import { RoutesParams } from '../../navigation/NavigationTypes'
+import createNavigationMock from '../../testing/createNavigationPropMock'
+import { getReturnOf } from '../../testing/helper'
+import { mockDisciplines } from '../../testing/mockDiscipline'
+import render from '../../testing/render'
+import DisciplineSelectionScreen from '../DisciplineSelectionScreen'
+
+jest.mock('@react-navigation/native')
+jest.mock('../../hooks/useLoadDisciplines')
+jest.mock('../../hooks/useReadSelectedProfessions')
```

Comment:
I think this can be removed.

Reply:
No the mocks are needed. The tests are failing without them